plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -531,7 +531,7 @@ func ClusterRoles() []rbacv1.ClusterRole {
 	nodeProxierRules := []rbacv1.PolicyRule{
 		rbacv1helpers.NewRule("list", "watch").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
 		rbacv1helpers.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
-
+		rbacv1helpers.NewRule("list", "watch").Groups(networkingGroup).Resources("servicecidrs").RuleOrDie(),
```

Comment:
Reminded by liggitt's comment in https://github.com/kubernetes/kubernetes/pull/122837#issuecomment-1898680505, adding this permission should be feature gated. service-cider-controller feature gate it: https://github.com/kubernetes/kubernetes/blob/8835dabc3bb38aaedaccf6a0237c1dabac17f0d8/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L372-L383

Reply:
will feature gate this then.